crates/bevy_reflect/bevy_reflect_derive/src/type_uuid.rs

```diff
@@ -17,9 +17,12 @@ pub(crate) fn type_uuid_derive(input: DeriveInput) -> syn::Result<TokenStream> {
             continue;
         };
 
-        let uuid_str = match &name_value.value {
```

Comment:
This expression became much less readable imho.

`uuid_str` is hidden inside a lot of syntax, and it is not immediately clear that it is the value being assigned.

Maybe disable lint just for this one line, or rewrite it to `let uuid_str = if (...) {} else {}` statement.

Reply:
Reverted this one, but for some reason the `allow` attribute only works when applied on the upper block level, not on the expression itself :confused: